batch-setup/make_meta_tiles.py

```diff
@@ -290,24 +472,65 @@ def _big_jobs(rawr_bucket, prefix, key_format_type, rawr_zoom, group_zoom,
 
     return big_jobs
 
+def viable_container_overrides(mem_mb):
+    """
+    Turns a number into the next highest even multiple that AWS will accept, and the min number of CPUs you need for that amount
+    :param mem_mb: (int) the megabytes of memory you'd request in an ideal world
+    :return: the amount of mem you need to request for AWS batch to honor it, the amount of vcpus you must request
+    """
+    mem_mb = int(mem_mb)
+
+    if mem_mb < 512:
+        return 512, 1
+
+    if mem_mb % 1024 == 0:
+        return mem_mb, 1
+
+    mem_gb_truncated = int(mem_mb / 1024)
+    next_gb = mem_gb_truncated + 1
+    desired_mem_mb = next_gb * 1024
+
+    max_mem_per_vcpu = 8 * 1024
+    vcpus = 1 + int((desired_mem_mb - 1)/max_mem_per_vcpu)
 
-def enqueue_tiles(config_file, tile_list_file, check_metatile_exists, mem_multiplier=1.0, mem_max=32 * 1024):
+    return desired_mem_mb, vcpus
+
+
+def enqueue_tiles(config_file, tile_list_file, check_metatile_exists, tile_specifier=TileSpecifier(),
+                  mem_multiplier=1.0, mem_max=32 * 1024):
     from tilequeue.command import make_config_from_argparse
     from tilequeue.command import tilequeue_batch_enqueue
     from make_rawr_tiles import BatchEnqueueArgs
 
-    args = BatchEnqueueArgs(config_file, None, tile_list_file, None)
     os.environ['TILEQUEUE__BATCH__CHECK-METATILE-EXISTS'] = (
         str(check_metatile_exists).lower())
-    with open(args.config) as fh:
+    with open(config_file) as fh:
         cfg = make_config_from_argparse(fh)
 
-    update_memory_request(cfg, mem_multiplier, mem_max)
-    tilequeue_batch_enqueue(cfg, args)
+    with open(tile_list_file, 'r') as tile_list:
+        coord_lines = [line.strip() for line in tile_list.readlines()]
+
+    reordered_lines = tile_specifier.reorder(coord_lines)
+
+    print("[%s] Starting to enqueue %d tile batches" % (time.ctime(), len(reordered_lines)))
+    for coord_line in reordered_lines:
+        # override memory requirements for this job with what the tile_specifier tells us
+        mem_mb = int(tile_specifier.get_mem_reqs_mb(coord_line))
+        adjusted_mem = mem_mb * mem_multiplier
 
+        # now that we know what we want, pick something AWS actually supports
+        viable_mem_request, required_min_cpus = viable_container_overrides(adjusted_mem)
+        print("REMOVEME: [%s] enqueueing %s at %s mem mb and %s cpus" % (time.ctime(), coord_line, viable_mem_request, required_min_cpus))
```

Comment:
Remove REMOVEME? This looks like a useful thing to print maybe.

Reply:
There are A LOT of these.  25k with this configuration